docs/rule/no-unnecessary-curly-strings.md

```diff
@@ -0,0 +1,29 @@
+# no-unnecessary-curly-strings
+
+Strings need not be wrapped in the mustache expressions.
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+<FooBar class={{"btn"}} />
+```
+
+```hbs
+<FooBar class="btn">{{"Hello"}}</FooBar>
+```
+
+This rule **allows** the following:
+
+```hbs
+<FooBar class="btn" />
+```
+
+```hbs
+<FooBar class="btn">Hello</FooBar>
+```
+
+## References
+
+* Ember's [Helper Functions](https://guides.emberjs.com/release/components/helper-functions/) guide
```

Comment:
I think this can go?

Reply:
Removed.

Reply:
It looks like we require one reference so I had to add back one generic link for now.